docs/topics/settings.rst

```diff
@@ -604,6 +604,26 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
+-----------------------
+
+Default: ``False``
+
+Whether or not to allow broken responses, where declared ``Content-Length``
+does not match content sent by the server. If ``False``, these responses
+trigger a ``ResponseFailed`` error. If ``True``, these responses are allowed
```

Comment:
I'm asking because the best default value for the option is not clear for me. False makes sense because it mimics existing behavior, True makes sense if the problem is usually caused by server misconfiguration, retry+True sounds like a compromise which could work better as a default in practice because it handles both cases, but I'm not really sure.